src/content/libraries/en/rita.js.yaml

```
@@ -0,0 +1,13 @@
+name: rita.js
+description: tools for natural language and generative writing
+category: language
+sourceUrl: https://github.com/dhowe/ritajs
+websiteUrl: https://rednoise.org/rita
+featuredImage: "../images/rita.jpg"
+author:
+  name: Daniel C Howe
+  url: https://rednoise.org/daniel
+featuredImageAlt: rita.js logo
+npm: rita
+npmFilePath: "https://unpkg.com/rita@3.1.3/dist/rita.min.js"
```

I think in this case you'll just need the bit after the version, similar to https://github.com/processing/p5.js-website/blob/df3d65019dafc24e54e8cb429cc9e38b1618c230/src/content/libraries/es/p5.xr.yaml#L12
```suggestion
npmFilePath: "dist/rita.min.js"
```